### PR TITLE
split MONDO:0014253

### LINKED ISSUE #4098: autoimmune lymphoproliferative syndrome type 3 ?
MONDO:0014253 autoimmune lymphoproliferative syndrome type 3 
I am wondering if the X-refs on this term are really equivalent? 
UMLS splits the X-refs across two CUIs and at first glance at the names, the only difference is Roman vs. Arabic numerals. But the NCI definition on Type 3 Autoimmune Lymphoproliferative Syndrome UMLS CUI: C1519711/ NCIT:C39577 that displays on the MONDO term is:  "A rare, primary immunodeficiency. It is caused by a currently _undetermined defect_ in the Fas-induced apoptosis pathway. No mutations in Fas, FASLG or CASP10 are detectable. Disruption of Fas-induced apoptosis impairs lymphocyte homeostasis and immune tolerance. Characteristic laboratory findings include an increase in circulating, double-negative (CD4-/CD8-) T cells in the setting of immune-mediated anemia, thrombocytopenia and neutropenia. Clinical signs present in childhood include fatigue, pallor, bruising, hepatosplenomegaly and chronic, non-malignant, non-infectious lymphadenopathy. The clinical course is influenced by a strong association with other autoimmune disorders and an increased risk for developing Hodgkin and non-Hodgkin lymphoma. [ NCIT:C39577 ]
But OMIM 615559 has specifically mapped the cause of "Autoimmune lymphoproliferative syndrome, type III" to variation at the PRKCD locus, and UMLS maps the OMIM term to UMLS CUI: C3809928. The Uniprot entry for this enzyme suggests it is involved in apoptosis: https://www.uniprot.org/uniprot/Q05655, so it could be the NCIT provided definition just needs to be updated?

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fixes #4098

### CODE REVIEW

**Comment by sabrinatoro** [comment]
@nicolevasilevsky : I don't understand why I get a proxy merge error. 
All the duplicate x-refs are on the obsolete term, and they have the "MONDO:obsoleteEquivalent" source. Does it still count as "proxy merge"?

**Comment by nicolevasilevsky** [comment]
I guess so - I guess you can't have an obsoleteEquivalent on an obsolete class and have it also be equivalentTo on a live class.

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
What should I do? Should I remove the xref from the obsoleted term? 
Is this something that should be discussed at a QC call?

**Comment by nicolevasilevsky** [comment]
Yeah, let's discuss on the QC call. I think we want to keep the xref on the obsoleted term, but I'm not 100% sure.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -280611,20 +280610,19 @@ synonym: "immunodeficiency, common variable, 9, formerly" RELATED [OMIM:615559]
 synonym: "PRKCD autoimmune lymphoproliferative syndrome" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
 synonym: "type 3 ALPS" EXACT [NCIT:C39577]
 synonym: "type 3 autoimmune lymphoproliferative syndrome" EXACT [NCIT:C39577]
-xref: DOID:0110119 {source="MONDO:equivalentTo"}
+xref: DOID:0110119 {source="MONDO:obsoleteEquivalent"}
 xref: ICD10CM:D47.9 {source="DOID:0110119"}
-xref: NCIT:C39577 {source="MONDO:equivalentTo"}
-xref: OMIM:615559 {source="DOID:0110119", source="MONDO:equivalentTo"}
+xref: NCIT:C39577 {source="MONDO:obsoleteEquivalent"}
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:280615]: For the record, we would use this in cases where we obsoleted the other source's class.

**Comment by sabrinatoro** [comment]
@nicolevasilevsky : I need help. I don't know why the checks failed.

**Comment by nicolevasilevsky** [comment]
I don't see what the issue is either, I think it may be something to do with these two classes, but I can't figure out what the issue is:
id: MONDO:800023
id: MONDO:800024

I am going to try to close and reopen, and that will retrigger the build, let's see if it happens again

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**